### DISCUSSION TITLE
For fields of type "File / Image / Video" provide an option to define the number of rows displayed

### DISCUSSION THREAD

**Discussion by robbiejackson** [pods-framework/pods#6759] [Ideas]
Enhancement suggestion submitted at the request of Jim True in response to [this](https://wordpress.org/support/topic/change-layout-for-photos-field-in-the-admin-edit-form-of-a-custom-post-type-pod/) support request. 

On the site I developed I’ve got a custom post type pod which is called ‘Walk’.
One of the fields of this Walk pod is called ‘photos’, and it’s a field of type “File / Image / Video”, allowing multiple files, up to a maximum of 6. So it’s used to link to uploaded photos that people have taken when they’ve been on the walk. 

When the administrator is editing these Walks, the photos appear in rows, with the thumbnail and the image title (used as a caption on the site) to the right of it. However, only 5 photos are shown, and he has to use the scroll bar to scroll down to see the sixth, as shown on the screen dump below. 

![photos](https://user-images.githubusercontent.com/5058681/34057535-3d6f8312-e1cf-11e7-9016-bb3ac1f8d608.jpg)

I did try the alternative “list style” of Tiles; that shows all 6 photos, but there is limited space for the caption, and for most of the captions only some of the text is shown, which makes it more difficult for editing.

It would be nice to have an option on the pods definition which specified the number of rows displayed on the admin form for editing the data for the pod. Or failing that, if the "max number of files" number is reasonably small (eg < 10), use that as the number of files to display. 

According to the site administrator, this issue has arised just recently, with a wordpress upgrade to 4.9.1.

**Comment by jimtrue** [comment]
@pglewis Is there a filter for this one to pick the number of rows to show in the list?
